android-exoplayer/src/main/java/com/brentvatne/exoplayer/MultiViewLayout.kt

```diff
@@ -0,0 +1,420 @@
+package com.brentvatne.exoplayer
+
+import android.content.Context
+import android.view.View
+import android.widget.FrameLayout
+import androidx.core.view.children
+import com.brentvatne.react.R
+import com.facebook.react.modules.i18nmanager.I18nUtil
+
+class MultiViewLayout(context: Context) : FrameLayout(context), MultiViewControlBar.MultiViewControlBarListener {
+
+    private val pipModeWindowScaleToParent = 3
+    private val isRTL = I18nUtil.getInstance().isRTL(context)
+    private val childSizeRatio: Float = 16f / 9f // width / height
+    private var swapChildViewPlayer = false
+    var gap: Int = 30 // pix
```

Comment:
can we pls move the constant into a `companion object`? 🙏

would be nice to find some better naming for `gap`, `multiViewLayoutGap` / `multiViewLayoutSpace` or something? also, shouldn't we use `dp` values instead of `px`?